test/NuGet.Core.Tests/NuGet.PackageManagement.Test/ProjectManagement/JsonConfigUtilityTests.cs

```diff
@@ -109,6 +109,59 @@ public void JsonConfigUtility_GetTargetFramework()
         }
 
         [Fact]
+        public void JsonConfigUtility_AddFramework()
+        {
+            // Arrange
+            var json = BasicConfig;
+
+            // Act
```

Comment:
nit: `Act` normally corresponds to a single line calling a method the test is written for. In this case it's line 121.